woocommerce/woocommerce-ios
# [Onboarding Deep Link] Improve unit test coverage for `AuthenticationManager`

Why
==========
Some unit test validation scenarios were missing from the deep link handling introduced in task #10843. This PR addresses this.

How
==========
Add URL validation failure scenario tests to the `AuthenticationManager`.

How to Test
==========
1. Just make sure CI is green, and the added unit tests make sense.

Update release notes:
